spec/support/features/session_helper.rb

```diff
@@ -489,7 +489,7 @@ def submit_form_with_valid_but_wrong_email
     end
 
     def click_link_to_use_a_different_email
-      click_link t('notices.use_diff_email.link')
+      click_link t('notices.use_diff_email.link').upcase_first
```

Comment:
Why was this necessary?

Reply:
This is a placeholder until the text content is finalized (new keys etc).

"click_link" is case sensitive.